concepticondata/conceptlists/Wittmann-1973-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	MAURITIAN	HAITIAN	FRENCH
+Wittmann-1973-200-1	1	98	ALL	all	tu	tu	tout
+Wittmann-1973-200-2	2	1577	AND	and	ek	é, épi	et
+Wittmann-1973-200-3	3	619	ANIMAL	animal	zanimo	bèt	animal
+Wittmann-1973-200-4	4	646	ASH	ashes	lasan	sân	cendres
+Wittmann-1973-200-5	5	1461	AT	at		a	à
+Wittmann-1973-200-6	6	1291	BACK	back	ledo	dó	dos
+Wittmann-1973-200-7	7	1292	BAD	bad	move	móvé	mauvais
+Wittmann-1973-200-8	8	1204	BARK	bark	lekors	ékòs	écorce
+Wittmann-1973-200-9	9	1157	BECAUSE	because	akoz	paské	parce que
+Wittmann-1973-200-10	10	1251	BELLY	belly	vât	vât	ventre
+Wittmann-1973-200-11	11	1202	BIG	big	gro, grâ, bel	gró, grâ	gros, grand
+Wittmann-1973-200-12	12	937	BIRD	bird	zozo	zwézó	oiseau
+Wittmann-1973-200-13	13	1403	BITE	to bite	mord-e	mòdé	mordre
+Wittmann-1973-200-14	14	163	BLACK	black	noar	nwa	noir
+Wittmann-1973-200-15	15	946	BLOOD	blood	disâ	sâ	sang
+Wittmann-1973-200-16	16	176	BLOW (WITH MOUTH)	to blow	sufle	suflé	souffler
```

Comment:
_souffler_ seems to be ambiguous. It is mapped to  BLOW (WITH MOUTH) and BLOW (OF WIND). If we take BLOW (OF WIND) as the prototypical one, I suggest to remap (see also #836).

Reply:
True, I forgot which of those was our prototypical case. Will change it.